im.dino.Dino.appdata.xml

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
there is alread an appdata file installed by upstream, you don't need this.

Reply:
That version lacks content_rating and release tag, that was the issue..

Reply:
that's done through patches. If there is an appdata file, it should be used.

Reply:
Okay.. I will patch it.